src/editor/editor.cpp

```diff
@@ -213,62 +214,44 @@ namespace Editor
             }
             ImGui::EndPopup();
         }
-        else if (ImGui::BeginPopupModal("TilesetLoadSuccess"))
-        {
-            ImGui::Text("Image loaded with height %i,\n width %i, channels %i!", tileset_height, tileset_width,
-                        tileset_channels);
-            ImGui::SetCursorPos(ImVec2{80, 50});
-            if (ImGui::Button("Ok", ImVec2(120, 0)))
-            {
-                ImGui::CloseCurrentPopup();
-            }
-            ImGui::EndPopup();
-        }
     }
 
     //--------------------------------------------------------
-    void
-    LoadTilesetPopupWindow()
+
+    void LoadTileSet()
     {
-        ImGui::SetNextWindowSize(window_size_popup);
-        ImGui::SetNextWindowPos(window_center_popup);
-        if (ImGui::BeginPopupModal("TilesetLoad"))
+        bool success = TilesetLoad();
+        
+        if (!success)
         {
-            ImGui::Text("Enter file name");
-            ImGui::Separator();
-            ImGui::PushStyleVar(ImGuiStyleVar_FramePadding, ImVec2(0, 0));
-            ImGui::InputText("File Name", tileset_file_path, IM_ARRAYSIZE(tileset_file_path));
-            ImGui::PopStyleVar();
-
-            if (ImGui::Button("Load", ImVec2(120, 0)))
-            {
-                bool success = TilesetLoad();
-                if (success)
-                {
-                    ImGui::OpenPopup("TilesetLoadSuccess");
-                }
-                else
-                {
-                    ImGui::OpenPopup("TilesetLoadError");
-                }
-            }
-
-            ImGui::SetItemDefaultFocus();
-            ImGui::SameLine();
-
-            if (ImGui::Button("Cancel", ImVec2(120, 0)))
-            {
-                ImGui::CloseCurrentPopup();
-            }
-
-            LoadTilesetImageResultPopupWindow();
-
-            ImGui::EndPopup();
+            ImGui::OpenPopup("TilesetLoadError");
         }
-
     }
 
+    //-----------------------------------------------------------
+    void 
+    LoadImageFromNativeFileDialog()
```

Comment:
It doesn't look like this function is loading an image, but rather selecting the image file name with the file dialog, so you probably want to change the function name to reflect that